src/nxt_http_route.c

```diff
@@ -1358,16 +1361,13 @@ nxt_http_pass_var(nxt_task_t *task, nxt_http_request_t *r,
 
 
 static void
-nxt_http_pass_query_ready(nxt_task_t *task, void *obj, void *data)
+nxt_http_pass_query(nxt_task_t *task, nxt_http_request_t *r,
+    nxt_http_action_t *action)
 {
-    nxt_int_t           ret;
-    nxt_router_conf_t   *rtcf;
-    nxt_http_action_t   *action;
-    nxt_http_status_t   status;
-    nxt_http_request_t  *r;
-
-    r = obj;
-    action = data;
+    nxt_int_t          ret;
+    nxt_router_conf_t  *rtcf;
+    nxt_http_status_t  status;
+
```

Comment:
Technically shouldn't `nxt_router_conf_t  *rtcf;` and `nxt_http_status_t  status;` be swapped around as when types are the same length we order alphabetically (yes it's the same order as before), not that I really care about this...

Reply:
Yep, it makes sense to consider the order. But I'd like to keep it the same as I touched it based on the existing order.